errbot/config-template.py

```diff
@@ -31,7 +31,16 @@
 # locally before publishing it. Note that you can specify only a single
 # directory, however you are free to create subdirectories with multiple
 # plugins inside this directory.
-BOT_EXTRA_PLUGIN_DIR = None 
+BOT_EXTRA_PLUGIN_DIR = None
+
+# If True, when you err wants to start a plugin and a dependency from
+# requirements.txt of the plugin is missing, it will automatically invoke
+# pip install --user for the missing dependencies.
+# It will detect if the bot is running in a virtualenv and if yes, it
+# will install it in the virtualenv instead of on the user.
+#
+# Note: be sure to give the proper autorizations to the user running the bot.
+AUTOINSTALL_DEPS = True
```

Comment:
I suggest changing this to the following text:

```
# Should plugin dependencies be installed automatically? If this is true
# then Err will use pip to install any missing dependencies automatically.
#
# If you have installed Err in a virtualenv, this will run the equivalent
# of `pip install`. If no virtualenv is detected, the equivalent of `pip
# install --user` is used to ensure the package(s) is/are only installed for
# the user running Err.
```